test/node-unit/cli/options.spec.js

```diff
@@ -526,7 +526,7 @@ describe('options', function () {
           loadOptions('--timeout 500'),
           'to have property',
           'timeout',
-          '500'
+          500
```

Comment:
[Non-Actionable] I suppose this is an unrelated change, but there's no reason why 500 should have been a string before. 👍 from me 😄